metric/asyncfloat64.go

```diff
@@ -207,7 +207,10 @@ type Float64Observer interface {
 	embedded.Float64Observer
 
 	// Observe records the float64 value.
-	Observe(value float64, opts ...ObserveOption)
+	//
+	// Use the WithAttributeSet (or, if performance is not a concern,
+	// the WithAttributes) option to record with attributes.
```

Comment:
This is more verbose, but I think it makes it more clear what's going on. Is the language correct, though?
```suggestion
	// the WithAttributes) option to attach the given attributes to the recorded value.
```

Reply:
I don't think the language is correct. Attributes are not attached to the recorded value in this process, the value and attributes together are a recorded measurement.

All instruments record measurement, not values. Currently, a measurement is defined as a value and attributes. When you specify the attributes options all you are the measurement attributes. When you do not provide the option, the measurement is the recorded value alone.

Reply:
I see. "[R]ecord with attributes" sounded too succinct to my ears, and took me a couple of readings to understand it, given the lack of the subject ("record _what_?").

Reply:
Maybe just change to "Use the WithAttributeSet (or, if performance is not a concern, the WithAttributes) option to record measurement attributes."?

Reply:
Yes, I like that. It makes sense to me immediately.

You could consider adding "as well" to the end, to make it clear that you're still recording what you would have recorded without supplying the option, plus a little more.

Reply:
"... to include measurement attributes"?

What about include instead of record?

Reply:
Updated. PTAL